simulator/config.yml

```diff
@@ -0,0 +1,45 @@
+# Configuration file for scheduler-simulator
```

Comment:
Sorry, I should've described more in the issue. 

In k8s, component configurations are often managed under version control.

For example
- [KubeSchedulerConfiguration](https://kubernetes.io/docs/reference/scheduling/config/)
- [descheduler's component config](https://github.com/kubernetes-sigs/descheduler/tree/master/pkg/apis/componentconfig).

So, what I expect is something like

```yaml
apiVersion: simulator.kube-scheduler-simulator.x-k8s.io/v1alpha1
kind: SimulatorConfiguration
port: 1212
etcd-url: "http://127.0.0.1:2379"
cors-allowed-origin-list:
  - "http://localhost:3000"
kube-config: ""
kube-api-host: "127.0.0.1"
kube-api-port: 3131
kube-scheduler-config-path: ""
external-import-enabled: false
external-scheduler-enabled: false
```

TBH I'm not so familiar with [code-generator](https://github.com/kubernetes/code-generator), but, I believe you only need to add some annotation on code like `k8s:deepcopy-gen` or `k8s:conversion-gen` etc and code-generator will generate most of codes.